src/chapters/02/main.tex

```diff
@@ -39,6 +39,7 @@ \section{Problem}\label{sec:problem}
 another barber's chair and barber.
 
 The mathematical tool used to model this situation is a Markov chain.
+%%% "The mathematical tool we will use to model this situation..." (This reads like it's the only tool to model this situation)
```

Comment:
To make sure we don't miss them (we can grep the whole directory) let's try and use `% TODO` as a single convention for notes in text.
```suggestion
% TODO "The mathematical tool we will use to model this situation..." (This reads like it's the only tool to model this situation)
```